diagrams/sequencediagram.txt

```diff
@@ -95,68 +93,89 @@ t_swap -> t_wallet : if Bob, SEND PENDING Msg Reveal (maker is sender)
 ==Changing semantics: On Commit-Reveal, Maker and Taker were the key roles. From now on Bob or Alice are the key roles. Now t_ is bob_ on the left and m_ is alice_ on the right.==
 ==Swap setup: Bob is left, Alice right==
 t_wallet -> t_database: Ctl CheckpointWalletBobPrelockBob
-t_wallet -> t_swap : Ctl CoreArbitratingSetup
+t_wallet -> t_swap : Msg CoreArbitratingSetup
+activate t_swap
```

Comment:
It creates the blocs along the swap daemons from `activate` to `deactivate` directive. I use them to signify what is the list of actions/messages done/sent upon above message reception (here the `Msg CoreArbitratingSetup`). IMO it helps because you know what should be in the fn that implements this handling.

I'm not attach to it, so we can remove them if we think it makes the diagram less readable.

Reply:
They are good, I just did not understand them previously.

Reply:
> It creates the blocs along the swap daemons from `activate` to `deactivate` directive. I use them to signify what is the list of actions/messages done/sent upon above message reception (here the `Msg CoreArbitratingSetup`). IMO it helps because you know what should be in the fn that implements this handling.
> 
> I'm not attach to it, so we can remove them if we think it makes the diagram less readable.

For new eyes that will wanna understand and modify this diagram consistent with changes to the implementation, I suggest adding this as a comment to the `.txt`.